modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/java/com/liferay/dynamic/data/mapping/web/internal/exportimport/data/handler/DDMStructureStagedModelDataHandler.java

```diff
@@ -544,6 +549,59 @@ private void _exportDDMFormLayout(
 			ddmFormLayoutPath, structureLayout.getDefinition());
 	}
 
+	private void _exportReferencedDDMStructure(
+		PortletDataContext portletDataContext, DDMStructure structure,
+		DDMFormField ddmFormField) {
+
+		long ddmStructureId = GetterUtil.getLong(
+			ddmFormField.getProperty("ddmStructureId"));
+
+		DDMStructure referencedStructure =
+			_ddmStructureLocalService.fetchDDMStructure(ddmStructureId);
+
+		if (referencedStructure == null) {
+			if (_log.isWarnEnabled()) {
+				_log.warn(
+					StringBundler.concat(
+						"Unable to find structure ", ddmStructureId,
+						" referenced in definition of structure ",
+						structure.getStructureId()));
+			}
+
+			return;
+		}
+
+		try {
+			StagedModelDataHandlerUtil.exportReferenceStagedModel(
+				portletDataContext, structure, referencedStructure,
+				PortletDataContext.REFERENCE_TYPE_DEPENDENCY);
+		}
+		catch (PortletDataException portletDataException) {
+			_log.error(
+				StringBundler.concat(
+					"Unable to export structure ",
+					referencedStructure.getStructureId(), " as a reference of ",
+					"structure ", structure.getStructureId()),
+				portletDataException);
+		}
+	}
+
+	private void _exportReferencedDDMStructures(
+		PortletDataContext portletDataContext, DDMStructure structure,
+		DDMForm ddmForm) {
```

Comment:
@mbowerman could you please sort this alphabetically?

Reply:
could you also rename from structure to ddmStructure? Thanks!

Reply:
@carolmariaabb Done